lib/studies/reply/adaptive_nav.dart

```diff
@@ -241,12 +241,23 @@ class _DesktopNavState extends State<_DesktopNav>
               );
             },
           ),
-          const VerticalDivider(thickness: 1, width: 1),
+          const VerticalDivider(thickness: 1, width: 2),
           Expanded(
-            child: _SharedAxisTransitionSwitcher(
-              defaultChild: _MailNavigator(
-                child: MailboxBody(key: widget.inboxKey),
-              ),
+            child: Row(
```

Comment:
I think you should be able to achieve this without using a Row. Perhaps replace the Row and Flexible widgets with a Center widget?

Reply:
Thanks for the suggestion that works!